### ISSUE TITLE
Can't access http://localhost:8888 after following the setup docker guide

### ISSUE THREAD

**Issue by MrChocolateBar** [OWASP/crAPI#15]
http://localhost:8025 is working and the docker image is up and running but i cant access the http:8888 path.

**Comment by gkarabotsos** [comment]
Same exact behavior on my end.

**Comment by piyushroshan** [comment]
Could you please provide logs till container dies.
`docker logs -f crapi-web`

**Comment by gkarabotsos** [comment]
Thank you for helping.
Its the `crapi-workshop` that times out.
```
--------------------------------------------------------
 docker-compose-wait 2.7.3
---------------------------
Starting with configuration:
 - Hosts to be waiting for: [crapi-community:8087,crapi-identity:8080,crapi-workshop:8000,mailhog:1025]
 - Timeout before failure: 240 seconds 
 - TCP connection timeout before retry: 5 seconds 
 - Sleeping time before checking for hosts availability: 0 seconds
 - Sleeping time once all hosts are available: 0 seconds
 - Sleeping time between retries: 5 seconds
--------------------------------------------------------
Checking availability of crapi-community:8087
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 not yet available...
Host crapi-community:8087 is now available!
--------------------------------------------------------
Checking availability of crapi-identity:8080
Host crapi-identity:8080 is now available!
--------------------------------------------------------
Checking availability of crapi-workshop:8000
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Host crapi-workshop:8000 not yet available...
Timeout! After 240 seconds some hosts are still not reachable
```

**Comment by piyushroshan** [comment]
Fixed in #16
